os/fs/driver/mtd/smart.c

```diff
@@ -18,7 +18,7 @@
 /****************************************************************************
```

Comment:
The path in the commit title need not be this specific, please take a look at other commit titles